mabl snippets/waitUntilNotPresent.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * TITLE: Wait until the provided element is no longer present up to 15 seconds
+ *        testing every 100 ms
+ *
+ * @mablParam: elementSelector - A CSS selector for the element
+ *
+ * @mablReturn: 'true' if the element disappeared within the timeout,
+ *              'false' otherwise
+ *
+ */
+async function mablJavaScriptStep(mablInputs, callback) {
+  const TIME_BETWEEN_MS = 100;
+  const TIMEOUT_SEC = 15;
```

Comment:
Maybe add a comment here about the max 30 seconds we will for the JS to return a value?

Reply:
I updated the comment before the loop and the header also contains this information